core/trino-main/src/main/java/io/trino/execution/DynamicFilterConfig.java

```diff
@@ -41,20 +41,20 @@
     private boolean enableCoordinatorDynamicFiltersDistribution = true;
     private boolean enableLargeDynamicFilters;
 
```

Comment:
Please add a code comment somewhere in this file which contains the explanation from the commit message about which limits concretely map to what.